.github/workflows/dapr.yml

```diff
@@ -98,6 +101,11 @@ jobs:
           docker -v
       - name: Parse release version and set REL_VERSION
         run: python ./.github/scripts/get_release_version.py
+      - name: Set DAPR_VERSION
```

Comment:
We do not need this step. get_release_version.py sets REL_VERSION. we can use this environment variable.

Reply:
Will remove. Our discussion from yesterday....the version is set to "edge" in the Makefile